src/main/scala/de/htwg/se/sudoku/model/fileIoComponent/fileIoSlickImpl/FileIO.scala

```diff
@@ -0,0 +1,122 @@
+package de.htwg.se.sudoku.model.fileIoComponent.fileIoSlickImpl
+
+import com.google.inject.{Guice, Inject}
+import com.google.inject.name.{Named, Names}
+import de.htwg.se.sudoku.{SlickModule}
```

Comment:
You can remove here the brackets because it's just one class to import (in my IDE I get a warning)...